tests/conftest.py

```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
Why no copyright here?

Reply:
Because there was no copyright :)
So far I remember, it was created by Tony so I didn't added Intel Copyrights.